<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>designer_gui</class>
 <widget class="QWidget" name="designer_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>917</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>designer_gui</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Movie database</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="movieListWidget">
     <property name="font">
      <font>
       <family>Microsoft YaHei Light</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Year of release : </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titleLineEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="nrOfLikesLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Trailer : </string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Number of likes : </string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="trailerLineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Title : </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="genreLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="releaseYearLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Genre : </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="font">
        <font>
         <family>HoloLens MDL2 Assets</family>
        </font>
       </property>
       <property name="text">
        <string>Delete movie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="font">
        <font>
         <family>HoloLens MDL2 Assets</family>
        </font>
       </property>
       <property name="text">
        <string>Add movie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="font">
        <font>
         <family>HoloLens MDL2 Assets</family>
        </font>
       </property>
       <property name="text">
        <string>Update movie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filterButton">
       <property name="font">
        <font>
         <family>HoloLens MDL2 Assets</family>
        </font>
       </property>
       <property name="text">
        <string>Filter movie</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Playlist</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="watchListWidget"/>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Input the preffered genre : </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="genrePlayListButton"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Play movie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removePlaylist">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Remove playlist</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="trailerRemove2"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
